src/modules/edc-demo/components/policy-view/policy-view.component.html

```diff
@@ -33,58 +33,73 @@
 
     <!-- Paginator -->
     <mat-paginator
+      *ngIf="policyList.isReady"
       hidePageSize="true"
       pageSize="10"
-      [length]="policyDefinitions?.length"></mat-paginator>
+      [length]="policyList.data.numTotalPolicies"></mat-paginator>
   </div>
 
-  <div fxLayout="column" fxLayoutAlign="space-around stretch">
-    <mat-card *ngFor="let policyDef of policyDefinitions" class="policy-card">
-      <mat-card-header>
-        <mat-icon mat-card-avatar>policy</mat-icon>
-        <mat-card-title
-          ><span class="code">{{ id(policyDef) }}</span></mat-card-title
-        >
-      </mat-card-header>
+  <div class="flex flex-row justify-center">
+    <edc-demo-loading-state
+      *ngIf="policyList.isLoading"></edc-demo-loading-state>
+    <edc-demo-error-state
+      *ngIf="policyList.isError"
+      [data]="policyList.error.failureMessage"></edc-demo-error-state>
+    <edc-demo-empty-state
+      *ngIf="policyList.isReady && policyList.data.numTotalPolicies === 0"
+      [data]="'NO POLICY FOUND'"></edc-demo-empty-state>
+    <div
+      *ngIf="policyList.isReady && policyList.data.numTotalPolicies >= 1"
```

Comment:
check which cardinalities are checked (in this entire page), fix number comparisons